backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/UserResolver.java

```diff
@@ -39,4 +40,13 @@ public List<ApiUserWithStatus> usersWithStatus() {
   public User user(@Argument UUID id) {
     return new User(_userService.getUser(id));
   }
+
+  @QueryMapping
+  public User userByLoginEmail(@Argument String loginEmail) {
+    try {
+      return new User(_userService.getUserByLoginEmail(loginEmail));
+    } catch (NonexistentUserException e) {
+      return null;
+    }
+  }
```

Comment:
I think it might be nicer to just add this argument/branching logic to the `user` query above.

Reply:
And that is just doing something like this

` user(id: ID, loginEmail: String): User @requiredPermissions(allOf: ["MANAGE_USERS"])
 `
 And then add checks at the UserResolver level to make sure that at least one of those parameters is passed along?

Reply:
> user(id: ID, loginEmail: String): User @requiredPermissions(allOf: ["MANAGE_USERS"])
And then add checks at the UserResolver level to make sure that at least one of those parameters is passed along?

I think just `email` would be fine for the name of the argument, but yes this is right. You could just do a simple if statement
```java
if(StringUtils.isBlank(email)){
      try {
      return new User(_userService.getUserByLoginEmail(loginEmail));
    } catch (NonexistentUserException e) {
      return null;
    }
}else{
    return new User(_userService.getUser(id));
}
```

Then on the `getUserByLoginEmail` method you could add an annotation to only allow super admins to call `@AuthorizationConfiguration.RequireGlobalAdminUser`